api/auth.js

```
@@ -0,0 +1,180 @@
+  return `${salt}:${derived.toString('hex')}`;
+}
+
+function verifyPassword(password, stored) {
+  const [salt, hash] = stored.split(':');
+  const derived = crypto.scryptSync(password, salt, SCRYPT_KEYLEN, SCRYPT_COST);
+  const expected = Buffer.from(hash, 'hex');
+  return crypto.timingSafeEqual(derived, expected);
+}
+
+// ─── JWT Utilities (reuses same secret as license tokens) ────────────────────
+if (!process.env.LICENSE_JWT_SECRET) {
```

ESLint

[error] 'process' is not defined.

(no-undef)